### ISSUE TITLE
List out all the countries in the dropdown

### ISSUE THREAD

**Issue by talhakpg** [talhakpg/Tashafi-Phase-2#3]
![image](https://user-images.githubusercontent.com/11936336/131661625-a01e2886-f3cd-489c-bc4a-319e35de97eb.png)

**Comment by talhakpg** [comment]
@hassanahak Please provide the list of countries in Arabic.

**Comment by mohi2021** [comment]
I have handed in the list of countries already @Aseem2345

**Comment by Aseem2345** [comment]
yes I add Text box there to add countries @hassanahak  is suggest regarding this on skype so  Add in one text box you add there country manually.
